test.py

```diff
@@ -27,8 +27,16 @@ def tearDown(self):
             self.assertEqual(self.memory, memory)
             torch.cuda.empty_cache()
 
-def run_model(model_class, model_path, device):
-    m = model_class(device=device)
+    def test_fx_profile(self):
```

Comment:
I guess this is a correctness test rather than something you're using to gather data.  Did you confirm if this gets invoked in CI as you wrote it? I think we run all tests from test.py.

Reply:
Correct, this is just a correctness test.  I verified it locally, I did not check the CI though.